general/devices.rst

```diff
@@ -0,0 +1,58 @@
+Devices
+==================
```

Comment:
What do you feel that we can do better with this page by having it in our documentation rather than at https://devices.ubports.com/#/ or https://ubports.com/page/get-ubuntu-touch ?

Reply:
I agree. The devices list should be at devices.ubports.com. It's not very tidy atm, but we'll fix it soon. A page like this in a wiki is often outdated and just generates work to maintain.

Reply:
As a newcomer I would think that all information is in documentation. If I need something I go there. Its just a common knowledge to go there. So we have to make https://devices.ubports.com page simple to find for everyone, or keep it in documentation. When I join UBports, I did not knew that page exists, I though I would find that information in documentation.

Reply:
IDK, I understand about having more work to maintain and I'm not for that.  But if we write something up, at least it is available for other groups to use as well.  If it is outdated, it can be stripped out.  If we have the people-power right now to create docs, lets use it.   Winter is coming and I will have more free time to help... give me something to do ;)